### PR TITLE
private/pedro/welcomedialog darkmode

### PR DESCRIPTION
**Pull request by pedropintosilva**
- Welcome: Refactoring: fix indentations and quotes
- WIP Pass ui-theme to welcome dialog

### CODE REVIEW

**Review by pedropintosilva — COMMENTED** [review]
@eszkadev can you help me here?

**Comment by eszkadev** [comment]
> @eszkadev can you help me here?

what exactly should I do?

**Comment by pedropintosilva** [comment]
Ah sorry if It wasn't clear. I was looking for code pointers based on the comments I have inserted. Or...aren't those github comments visible to you?

File: browser/welcome/welcome.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 
-<html>
+<html data-theme="%UI_THEME%">
```
Comment (eszkadev) [inline browser/welcome/welcome.html:3]: this will not work as welcome.html isn't processed like  the main app, so variable will not be replaced

Comment (eszkadev) [reply browser/welcome/welcome.html:3]: change would be nedded in: `FileServerRequestHandler::preprocessWelcomeFile`

**Comment by eszkadev** [comment]
I think easiest is to add:     `&ui_theme=xxx` to the url and then replace %UI_THEME in preprocessing c++ code based on that param

**Comment by pedropintosilva** [comment]
@eszkadev well passing things to the url was exactly how I did it in the past https://github.com/CollaboraOnline/online/commit/90cc39061859ab7229b3b42d34305e0f0080d460 (see the before diff) but now all the code is different and I have not clue on what's the best to achieve the fix that is needed

**Comment by pedropintosilva** [comment]
tested, and added CSS bits, should be merged

**Pull request merged**